person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: 'Unknown')
+    @id = Ramdom.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission == 'allowed'
+  end
```

Comment:
- [ ] Kindly note that the logic behind this method `can_use_services?` is that it should return true if the @age >= 18  or if they have permission from parents(`@parent_permission == true`).
```rb
def can_use_services? 
    of_age? || @parent_permission
  end
```

Reply:
This issue is well resolved! ✔️